rgeo-shapefile.gemspec

```diff
@@ -22,4 +22,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "minitest", "~> 5.3"
   spec.add_development_dependency "rake", "~> 12.0"
+  spec.metadata["rubygems_mfa_required"] = "true"
```

Comment:
I would rather put this with the first block (e.g. after `spec.license` for instance)